From 79c99b3bc98b0da2b2bd6c7bcb1798017f15c897 Mon Sep 17 00:00:00 2001
From: Alison Wang <alison.wang@nxp.com>
Date: Fri, 31 May 2019 14:57:41 +0800
Subject: [PATCH 4/6] imx6qsabre: spl: Add mmc write support

This patch adds mmc write support in SPL for I.MX6QSABRE board.

Signed-off-by: Alison Wang <alison.wang@nxp.com>
---
 configs/mx6sabresd_defconfig | 1 +
 1 file changed, 1 insertion(+)

diff --git a/configs/mx6sabresd_defconfig b/configs/mx6sabresd_defconfig
index e81dc8a..2c643ad 100644
--- a/configs/mx6sabresd_defconfig
+++ b/configs/mx6sabresd_defconfig
@@ -7,6 +7,7 @@ CONFIG_SPL_LIBGENERIC_SUPPORT=y
 CONFIG_SYS_MALLOC_F_LEN=0x4000
 CONFIG_TARGET_MX6SABRESD=y
 CONFIG_SPL_MMC_SUPPORT=y
+CONFIG_SPL_MMC_WRITE=y
 CONFIG_SPL_SERIAL_SUPPORT=y
 CONFIG_NR_DRAM_BANKS=1
 CONFIG_SPL=y
-- 
2.7.4

